extensions/SemanticMediaWiki/includes/specials/SMW_SpecialSMWAdmin.php

```diff
@@ -84,6 +85,8 @@ public function execute( $par ) {
 		} else {
 			$refreshjob = null;
 		}
+		**/
+		$refreshjob = null; // Wikia change - we do not use job DB table for handling tasks queue (SUS-1305)
 
 		/**** Execute actions if any ****/
 		switch ( $this->getRequest()->getText( 'action' ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "action" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)